apps/docs/src/pages/components/stack.mdx

```diff
@@ -0,0 +1,99 @@
+import { HvStack, stackClasses, theme } from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("Stack", "core", stackClasses);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Playground
+  Component={HvStack}
+  componentName="HvStack"
+  controls={{
+    direction: {
+      defaultValue: "row",
+    },
+    spacing: {
+      defaultValue: "sm",
+    },
+    divider: {
+      defaultValue: false,
+    },
+  }}
+>
+
+    {Array.from({ length: 4 }).map((_, index) => (
```

Comment:
not a big deal, but `Array(n).keys()` is shorter, faster, and you get the index as values 😄 

https://jsperf.app/goqema/1/preview